### PR TITLE
fix(r-c-utils): update for federation

### PR DESCRIPTION
**Pull request by mwegman**
- update validateAndDecodeId to use SDK
- use 'url' to load previous messages

### CODE REVIEW

File: packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.js [hunk 1]
```diff
@@ -33,31 +33,31 @@ export function getConvoIdFromUrl(convoUrl) {
...
-      resultId = base64uuid;
+  }
+  catch (e) {
+    if (e.name !== 'InvalidCharacterError') {
+      throw e;
     }
   }
 
-  return resultId;
+  return {};
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.js]: This is no longer returning a false when not verified. Does this affect anything using it downstream? 

Also, could you update the jsdoc to match what the new return value is?

File: packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    it('deconstructs a hydra ID', () => {
+      const id = uuid.v4();
+      const cluster = 'us';
+      const hydraId = buildHydraRoomId(id, cluster);
+
+      expect(validateAndDecodeId(hydraId)).toEqual({id, cluster, type: hydraTypes.ROOM});
+    });
+
+    [null, '', 'AB%CD']
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.test.js]: Could you separate these into three `it` blocks? We forgot to mention it during our scrum meetings, but I believe we didn't want the test looping (am I correct @lalli-flores?).

File: packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import uuid from 'uuid';
+import {buildHydraRoomId, hydraTypes} from '@webex/common';
+
+import {validateAndDecodeId} from './uuid-utils';
+
+describe('uuid-utils', () => {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.test.js]: Thanks for adding these!

File: packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.js (hunk 1 above)
Comment (mwegman) [reply packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.js]: The rest of this PR are the changes necessary to move from 'false' to an empty object.

**Pull request merged**